system_files/just/custom.just

```diff
@@ -10,6 +10,9 @@ framework-13:
       rpm-ostree kargs --append="module_blacklist=hid_sensor_hub"
     fi
 
+framework-13-amd:
+#  rpm-ostree kargs --append=""
```

Comment:
```suggestion
   rpm-ostree kargs --append="cpufreq.default_governor=powersave" --append="pcie_aspm.policy=powersupersave" --append="amd_pstate=active"
```

These are some suggestions from the framework community discourse and also from https://github.com/achhabra2/fw13-amd-kinoite/blob/live/config/files/usr/share/ublue-os/just/60-custom.just

Reply:
I disagree with these:

`amd_pstate=active` is unnecessary on kernel 6.5 and newer.  It's already the default behavior unless the distro had changed the kernel defaults.

`cpufreq.default_governor=powersave` is going to break the ability to go into performance mode for the CPU in general.  I think it's better to adopt https://gitlab.freedesktop.org/upower/power-profiles-daemon/-/merge_requests/129

`pcie_aspm.policy=powersupersave` May be too aggressive for ASPM policy on a system that can have WLAN and SSD swapped out.  IMO If there is a problem with the BIOS defaults, it's better to get Framework to fix the BIOS defaults, validate it in their lab against all hardware they support and then push a BIOS upgrade.

Reply:
Just follow what @superm1, thanks for the huge explainers!